packages/contractkit/src/wrappers/Reserve.ts

```diff
@@ -0,0 +1,10 @@
+import { Reserve } from '../generated/types/Reserve'
+import { BaseWrapper, proxyCall, toBigNumber } from './BaseWrapper'
+
+export class ReserveWrapper extends BaseWrapper<Reserve> {
+  tobinTaxStalenessThreshold = proxyCall(
```

Comment:
Add [typedoc comments](https://typedoc.org/guides/doccomments/)

Reply:
The wrappers don't seem to have much comments  (perhaps because many would be the same as in the contract), but I'll at least add short descriptions of each contract.

Reply:
You can copy docs from the contract. But follow typedoc conventions

There's a branch where i've added docs for all methods on the wrappers

Reply:
The idea is that we will generate docs from the .ts files and expose them in a public page.

Reply:
Tried to add comments to all wrappers that I added.